src/main/java/io/pravega/connectors/flink/util/FlinkPravegaParams.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.connectors.flink.util;
+
+import com.google.common.collect.Sets;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.connectors.flink.FlinkPravegaReader;
+import io.pravega.connectors.flink.FlinkPravegaWriter;
+import io.pravega.connectors.flink.PravegaEventRouter;
+import io.pravega.connectors.flink.serialization.PravegaSerialization;
+import java.io.Serializable;
+import java.net.URI;
+import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.streaming.util.serialization.DeserializationSchema;
+import org.apache.flink.streaming.util.serialization.SerializationSchema;
+
+/**
+ * Convenience class for extracting pravega parameters from flink job parameters.
+ *
+ * As a convention, the pravega controller uri will be passed as a parameter named 'controller'.
+ *
+ * Operations are provided to create streams, readers and writers based on stream names as
+ * parameters.
+ *
+ * @see StreamInfo
+ */
+public class FlinkPravegaParams {
+    public static final String DEFAULT_CONTROLLER_URI = "tcp://127.0.0.1:9090";
+    public static final String CONTROLLER_PARAM_NAME = "controller";
+
+    private ParameterTool params;
+
+    public FlinkPravegaParams(ParameterTool params) {
+        this.params = params;
+    }
+
+    /**
+     * Gets the controller URI from the 'controller' job parameter. If this parameter
+     * is not specified this defaults to DEFAULT_CONTROLLER_URI.
+     */
+    public URI getControllerUri() {
+        return URI.create(params.get(CONTROLLER_PARAM_NAME, DEFAULT_CONTROLLER_URI));
+    }
+
+    /**
+     * Gets a stream information from a stream specification.
```

Comment:
The javadoc here is a bit confusing, it gets stream information from a job parameter.  Maybe `streamParam` should be `streamParamName` too.